docs/zk2kraft.html

```diff
@@ -176,5 +176,16 @@ <h5 class="anchor-heading">Removal metrics</h5>
                 Kafka remove all zookeeper dependencies, so the metrics is removed.
             </p>
         </li>
+        <li>
+            <p>
+                Remove the metrics which is monitoring requests waiting in the topic purgatory
+            </p>
+            <ul>
+                <li><code>kafka.server:type=DelayedOperationPurgatory,name=PurgatorySize,delayedOperation=topic</code></li>
```

Comment:
I had a look at the `DelayedOperationPurgatory` class and I believe you are actually removing 2 metrics instead of one
```
metricsGroup.newGauge("PurgatorySize", this::watched, metricsTags);
metricsGroup.newGauge("NumDelayedOperations", this::numDelayed, metricsTags);
```
Could you add this to the description as well?

Reply:
Thanks for your review, addressed it.